teamsupport/models.py

```diff
@@ -23,14 +35,41 @@ def __init__(self, client, ticket_id=None, data=None):
                 '(neither given)')
         self.id = self.TicketID
 
+    @classmethod
+    def create(cls, first_name, email, category, name, description):
```

Comment:
We have only "First Name" on our "Contact Support" form. FirstName is a separate attribute of Contact Details in TeamSupport API. So, I left it as is.